src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -104,29 +104,14 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
 
     /**
      * Updates the feedback question identified by `{@code newAttributes.getId()}
-     *   and changes the {@code updatedAt} timestamp to be the time of update.
      * For the remaining parameters, the existing value is preserved
      *   if the parameter is null (due to 'keep existing' policy).<br>
-     *
-     * <p>Preconditions:
-     * {@code newAttributes.getId()} is non-null and correspond to an existing feedback question.
-     */
-    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes)
-            throws InvalidParametersException, EntityDoesNotExistException {
-        updateFeedbackQuestion(newAttributes, false);
-    }
-
-    /**
-     * Updates the feedback question identified by `{@code newAttributes.getId()}
-     * For the remaining parameters, the existing value is preserved
-     *   if the parameter is null (due to 'keep existing' policy).<br>
-     * The timestamp for {@code updatedAt} is independent of the {@code newAttributes}
-     *   and depends on the value of {@code keepUpdateTimestamp}
+     * The timestamp for {@code updatedAt} is dependent of the {@code newAttributes}
```

Comment:
`The timestamp for {@code updatedAt} is dependent of the {@code newAttributes}` this is not actually true. You should remove it. The `updatedAt` is updated by the Objectify library.

Reply:
Maybe `The timestamp for {@code updatedAt} will be updated` would be better. There is no dependency here.

Reply:
sorry, I should have done that in the first place I made a pull request again with the changes, it will be ready to review once the CI tests are passed.